mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/mosaic.kt

```diff
@@ -92,10 +95,35 @@ public suspend fun runMosaic(body: suspend MosaicScope.() -> Unit): Unit = corou
 		}
 	}
 
+	val terminal = Terminal()
+	val terminalInfo = mutableStateOf(
+		TerminalInfo(
+			size = TerminalInfo.Size(terminal.info.width, terminal.info.height)
+		)
+	)
+
+	launch(context = composeContext) {
+		while (true) {
+			if (terminal.info.updateTerminalSize()
+				&& (terminalInfo.value.size.width != terminal.info.width
+					|| terminalInfo.value.size.height != terminal.info.height)
+			) {
+				terminalInfo.value = terminalInfo.value.copy(
```

Comment:
```suggestion
      val currentTerminalInfo = terminalInfo.value
			if (terminal.info.updateTerminalSize()
				&& (currentTerminalInfo.size.width != terminal.info.width
					|| currentTerminalInfo.size.height != terminal.info.height)
			) {
				terminalInfo.value = currentTerminalInfo.copy(
```
Multiple calls to `.value` cause more work than one might imagine to occur. Doing it once will save that work, especially for something happening 20 times per second.

Reply:
Oh, thanks, I didn't think(